.npmignore

```diff
@@ -1,9 +1,14 @@
 /.*
 
+/bin/
+/chrome.json
+/renovate.json
+
 /bower_components/
 
 /coverage/
 /demos/
+/src/
```

Comment:
On a general note it is a very bad idea. This means consumers of the library cannot build and do all the optimisations they potentially could. 
If you ever publish any library please always include sources in the final published package